src/error_gui.cpp

```diff
@@ -113,8 +113,15 @@ struct ErrmsgWindow : public Window, ErrorMessageData {
 	{
 		this->InitNested();
 
+		Colours window_colour = COLOUR_RED;
+		if (level == WL_INFO) window_colour = COLOUR_GREEN;
+		if (level == WL_WARNING) window_colour = COLOUR_ORANGE;
+		this->GetWidget<NWidgetCore>(WID_EM_CAPTION)->colour = window_colour;
+		this->GetWidget<NWidgetCore>(WID_EM_CLOSEBOX)->colour = window_colour;
+		this->GetWidget<NWidgetCore>(WID_EM_PANEL)->colour = window_colour;
```

Comment:
This should use the same background/foreground colours as `ShowGoalQuestion`. (I have no preference of which of them to adjust)